docs/concepts/snapshot.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Snapshot Concept
+headline: Snapshot
+bodyclass: docs
+layout: docs
+sidenav: doc-side-concepts-nav.html
+type: markdown
+---
+<h2 class="top">Snapshot</h2> 
+
+Snapshot is a current state of an Aggregate.
```

Comment:
Not necessarily ”current”. It may be a state of the aggregate from the past. A snapshot ”sits” in between events in the history, so that you don't have to replay them all. You read events backwards until you encounter a snapshot, then apply it to the aggregate, and then play trailing events.